### PR TITLE
New utility script util/load_sku_packs.py

### PR DESCRIPTION
**Pull request by gpaulos**
Back by popular demand, I have updated utility script to load SKU packs to the RackHD host to work with the new config system.

This script is based upon the routine used in the stack init script but its broken down into three steps. 

@hohene @jimturnquist @johren

### CODE REVIEW

File: test/util/load_sku_packs.py [hunk 1]
```diff
@@ -35,40 +39,38 @@ def test01_preload_sku_packs(self):
                     subprocess.call("cd on-skupack; ./build-package.bash "
                                     + skus + " " + skus + " >/dev/null 2>&1", shell=True)
             break
```
Comment (hohene) [inline test/util/load_sku_packs.py]: Can you break the "build sku packs" subsection into a separate test too?  When using the script, I'll pull down the sku packs repo, modify some data in an existing sku pack or create a new one, then call your build code chunk.    Then I can keep calling the build routine....without downloading again.

File: test/util/load_sku_packs.py [hunk 2]
```diff
@@ -35,40 +39,38 @@ def test01_preload_sku_packs(self):
...
-        # upload SKU packs to ORA
-        print "**** Loading SKU Packs to server"
+
+    def test02_upload_sku_packs(self):
+        # upload SKU packs to RackHD
         for subdir, dirs, files in os.walk('on-skupack/tarballs'):
```
Comment (hohene) [inline test/util/load_sku_packs.py]: do we need a quick check to verify if the directory exists before walking it?

File: test/util/load_sku_packs.py [hunk 3]
```diff
@@ -35,40 +39,38 @@ def test01_preload_sku_packs(self):
...
                                      payload=file(fit_common.TEST_PATH + "on-skupack/tarballs/" + skupacks).read())
             break
         print "\n"
+
+    def test03_verify_sku_packs(self):
         # check SKU directory against source files
-        errorcount = 0
+        errorcount = ""
         skulist = fit_common.json.dumps(fit_common.rackhdapi("/api/2.0/skus")['json'])
         for subdir, dirs, files in os.walk('on-skupack'):
```
Comment (hohene) [inline test/util/load_sku_packs.py]: same comment, quick check if on-skupack directory exists?

File: test/util/load_sku_packs.py [hunk 4]
```diff
@@ -2,27 +2,31 @@
...
 # set path to common libraries
 sys.path.append(subprocess.check_output("git rev-parse --show-toplevel", shell=True).rstrip("\n") + "/test/common")
 import fit_common
 
-class load_sku_packs(fit_common.unittest.TestCase):
-    def test01_preload_sku_packs(self):
-        print "**** Processing SKU Packs"
-        # Load SKU packs from GutHub
+# Select test group here using @attr, these can be any labels to run groups of tests selectively
+# @attr is a decorator and mus be located in the line just above the class to be labeled
```
Comment (yyscamper) [inline test/util/load_sku_packs.py]: a little typo: mus -> must

File: test/util/load_sku_packs.py [hunk 5]
```diff
@@ -2,27 +2,31 @@
...
+from nose.plugins.attrib import attr
+@attr(all=True, regression=True, smoke=True)
+
+class fit_template(fit_common.unittest.TestCase):
+
+    def test01_download_sku_packs(self):
+        # Download SKU packs from GutHub
```
Comment (yyscamper) [inline test/util/load_sku_packs.py]: typo: guthub => github

File: test/util/load_sku_packs.py [hunk 6]
```diff
@@ -35,40 +39,38 @@ def test01_preload_sku_packs(self):
...
-                fit_common.rackhdapi("/api/1.1/skus/pack", action="binary-post",
+                fit_common.rackhdapi("/api/2.0/skus/pack", action="binary-post",
                                      payload=file(fit_common.TEST_PATH + "on-skupack/tarballs/" + skupacks).read())
             break
         print "\n"
+
+    def test03_verify_sku_packs(self):
         # check SKU directory against source files
-        errorcount = 0
+        errorcount = ""
```
Comment (yyscamper) [inline test/util/load_sku_packs.py]: Please rename the `errorcount` to something like `error_message` as it is not a number

File: test/util/load_sku_packs.py (hunk 2 above)
Comment (gpaulos) [inline test/util/load_sku_packs.py]: No, the directory is explicitly created

File: test/util/load_sku_packs.py (hunk 3 above)
Comment (gpaulos) [inline test/util/load_sku_packs.py]: No, the directory is explicitly created

File: test/util/load_sku_packs.py (hunk 4 above)
Comment (gpaulos) [inline test/util/load_sku_packs.py]: Yep, will remove that whole comment. That came from the template.

File: test/util/load_sku_packs.py (hunk 5 above)
Comment (gpaulos) [inline test/util/load_sku_packs.py]: Yep, will fix.

File: test/util/load_sku_packs.py (hunk 6 above)
Comment (gpaulos) [inline test/util/load_sku_packs.py]: OK, good idea.

File: test/util/load_sku_packs.py (hunk 1 above)
Comment (gpaulos) [inline test/util/load_sku_packs.py]: Sure

**Comment by gpaulos** [comment]
Had to add one more change to conform to PR#583. This script will be broken until that PR is merged.

File: test/util/load_sku_packs.py [hunk 7]
```diff
@@ -2,30 +2,34 @@
...
+
+class fit_template(fit_common.unittest.TestCase):
+
+    def test01_download_sku_packs(self):
+        # Download SKU packs from GitHub
         subprocess.call("rm -rf temp.sku; rm -rf on-skupack", shell=True)
         os.mkdir("on-skupack")
         # download all SKU repos and merge into on-skupack
-        for url in fit_common.GLOBAL_CONFIG['repos']['skupack']:
+        for url in fit_common.fitskupacks():
```
Comment (hohene) [inline test/util/load_sku_packs.py]: remove the 's' from fit_common.fitskupacks(), should be fit_common.fitskupack()

**Comment by gpaulos** [comment]
Going to close and resubmit due to conflicts

**Pull request closed without merging**